Applications/Spire/Include/Spire/SecurityInput/LocalSecurityInputModel.hpp

```diff
@@ -25,6 +26,8 @@ namespace Spire {
     private:
       rtv::Trie<char, Nexus::SecurityInfo> m_securities;
   };
+
+  Beam::Ref<const LocalSecurityInputModel> get_local_security_input_test_model();
```

Comment:
```Ref``` is used as a parameter to indicate lack of ownership, similar to things like ```std::string_view```. It should not be used as a return type.